CHANGES.rst

```diff
@@ -12,6 +12,11 @@ lib
 
 - Make EngDB_Value public for JSDP use [#5663]
 
+ramp_fitting
+------------
+
+- Update documentation to define optimal weighting algorithm [#5300]
```

Comment:
Actual PR number is 5676, not 5300.

Reply:
May or may not have been listing issue numbers in the changelog instead of PR numbers... 😬 fixing now.